test/support/helpers/businesses/hiring_invoice_requests_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Admin
+  module Businesses
+    module HiringInvoiceRequestsHelper
```

Comment:
We can remove the `Admin` module/namespace here.

Reply:
module removed in 93591ef79cae96bd3aaba88b5089f04867120fec